docs/source/loading.rst

```diff
@@ -0,0 +1,304 @@
+Load
+====
+
+You have already seen how you can load a dataset from the Datasets Hub. But datasets are stored in a variety of places, and sometimes you won't find the one you want on the Hub. The datasets can be on your local machine, in a Github repository, and in data structures like Python dictionaries and Pandas DataFrames. Wherever your dataset may be stored, Datasets provides a way for you to load it and use it for training.
+
+This guides shows you how to load a dataset from:
+
+* local files
+* in-memory data
+* in streaming mode
+* offline
+* a specific slice of a split
+
+Local files
+-----------
+
+Datasets can be loaded from local files stored on your computer. The datasets most likely exist as a ``csv``, ``json`` or ``txt`` file. Datasets provides the :func:`datasets.load_dataset` method for loading these file types.
```

Comment:
We need to add `parquet` here as well, and have a subsection for parquet too like CSV or JSON

Reply:
can i use the code sample from #2733 for the parquet subsection?